RELEASING.md

```diff
@@ -0,0 +1,30 @@
+# Releasing
+
+This document outlines the automated process for releasing new versions of Misk repository.
```

Comment:
Releases are automatic, should this doc be written more assuming that. It initially comes off as that this is a more manual process.

Reply:
Good suggestion! I made some changes, please take a look and feel free to update. Thanks!